docs/app/pages/components/components-sections/side-navigation/navigation/snippets/app.html

```diff
@@ -0,0 +1,34 @@
+<ux-side-navigation style="width: 750px">
```

Comment:
Please use the hierarchical example from http://m.everythingux.net/styleguide/ui-components/side-menus/ here. This is implemented in the MF documentation, for reference.